cloudinit/distros/arch.py

```diff
@@ -62,9 +62,9 @@ def install_packages(self, pkglist):
         self.update_package_sources()
         self.package_command('', pkgs=pkglist)
 
-    def _write_network_config(self, netconfig):
+    def _write_network_state(self, network_state):
```

Comment:
I think we should also  drop "_bring_up_interfaces" from arch as it as a near equivalent (yet less functional) of the parent class `Distro._bring_up_interfaces`